ModeratorFrontEnd/README.md

```diff
@@ -0,0 +1,73 @@
+## AI For Orcas - Moderator Front End System
+
+The moderator front in is responsible for presenting the candidate whale calls to the designated moderators so they can confirm the calls as legitimate or as false positives.
+
+The basic workflow is as follows:
+- The notification system sends designated moderators an email indicating there are one or more candidates to review.
+- The moderator logs into the moderator front end and is presented with the list of candidate calls including a spectrogram image of the call.
+- The moderator then listens to these calls and indicates whether it is a confirmed call, a false positive, or an undetermined call; they can also add sortable tags and provide comments.
+- If the call is confirmed, notifications are sent out to the designated subscribers informing them of the presence of whales in the designated area.
+
+## Architecture
+The moderator front end solution has two primary components that are both located in the /ModeratorFrontEnd/AIForOrcas folder:
+
+### A Web API to Interact with the CosmosDB backend
+This is a standard .NET Core Web API found in the AIForOrcas.Server project and is published to Azure as https://aifororcasdetections.azurewebsites.net
+
+#### Publish Settings
+- Site Url: https://aifororcasdetections.azurewebsites.net
+- Resource group: AIForOrcas
+- Configuration: Release
+- Target framework: netcoreapp3.1
+- Deployment mode: Self-contained
+- Target runtime: win-x86
+
+### A Web Fontend
+This is a .NET Core Blazor-based web site found in the AIForOrcas.Client.Web project and is published to Azure as https://aifororcas.azurewebsites.net
+
+#### Publish Settings
+- Site Url: https://aifororcas.azurewebsites.net
+- Resource group: AIForOrcas
+- Configuration: Release
+- Target framework: netcoreapp3.1
+- Deployment mode: Self-contained
+- Target runtime: win-x86
+
+### Additional Projects
+The remaining projects in the solution represent code that is shared between the two projects (i.e. DTOs/Models, Business Logic, etc.)
+
+## Release Notes
+
+### 2021-03-15
```

Comment:
@micowan - appreciate you cleaning up the dev process with this readme and release notes. 

How do you feel about starting to officially version releases as below? 

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository 
https://co-pilot.dev/changelog

Feel free to close / resolve if you think this is better done independent of this PR.

Reply:
Does not really matter to me one way or the other. I was asked to update the README.md so I did. If we want to switch to the method you suggested, we can certainly start doing that. A call for the team I think.